processes/releases.md

```diff
@@ -0,0 +1,42 @@
+# New MeiliSearch releases - what to do
+
+Those are the steps we need to take every time we release a new version of MeiliSearch.
+
+## Responsibilities
+
+Right now, unless mentioned, the tasks for a new release on the developer relations side are divided in three categories or actions with each person accountable for them:
+- Documentation & guides: [@react-learner](https://github.com/react-learner)
+- Blog posts: [@CaroFG](https://github.com/CaroFG)
+- Social media: [@fharper](https://github.com/fharper)
+
+## After the last release
+
+1. Create a PR on a branch called prepare-meilisearch-[YYYY]-[WW] (i.e.: prepare-meilisearch-2020-03 for the third week of the year 2020) in the [documentation repository](https://github.com/meilisearch/documentation).
+1. Ensure that all new PRs related to the next release are done on the mentioned above.
+1. The documentation must contain at minimum the references of the new feature(s), if any.
+1. The decision should be made if we need to create a [guide](https://docs.meilisearch.com/guides/) for the new feature(s), if any.
+
+## Pre-release week
+
+1. If the release has substantial new features, we should draft a [blog](https://blog.meilisearch.com/) post about the new release.
+1. The prepare-meilisearch-[YYYY]-[WW] should be reviewed and ready to merge to the [master branch](https://github.com/meilisearch/documentation/tree/master).
+
+## Release day
+
+1. The prepare-meilisearch-[YYYY]-[WW] branch should be merged with the [master brand](https://github.com/meilisearch/documentation/tree/master), and updated version of the documentation published [in production](https://docs.meilisearch.com/).
+1. Once in production, we should publish the blog post, if any.
```

Comment:
```suggestion
2. Once in production, we should publish the blog post, if any.
```

Reply:
@CaroFG : maybe I'm missing something, but there is no change in this suggestion?

Reply:
Oh I get it, with Markdown, you can do ordered list with only 1. for each items, and the result generated will put the right numbers. I prefer to do it like that, so I don't have to change the number if I add items or reorganized them. I put specific numbers after list that were splited so the generate list continue at the right number.